build/constants.yaml

```diff
@@ -108,6 +109,8 @@
       directory: src/startupscriptgenerator/src/common/consts
 - name: node-versions
   constants:
+    node-runtime-base-tag: 20191023.4
```

Comment:
is there a way to avoid repeating this value here (from `build/__nodeVersions.sh `)

Reply:
`__nodeVersions.sh` is generated from this .yaml

Reply:
You mean repeating between this line here and the next (yarn-cache-base-tag)? I wanted it to be flexible here because sometimes we have seen these tag numbers being different between different platforms depending on how successful a build went. For example, we could be having a failure in building Node runtime images but could have been successful in case of Yarn in which case we should be able to use the tag for Yarn cache separately